data/human/news.txt

```diff
@@ -2125,6 +2315,9 @@ news "safety posters"
 			"KEEP FINGERS CLEAR OF GRINDER"
 			"REPORT ALL WORKPLACE INJURIES"
 			"PROTECTIVE EQUIPMENT REQUIRED WITHIN HAZARD ZONE"
+			"Keep hands on railing when traversing stairs."
```

Comment:
Should these also be all caps given the other ones are?

Reply:
They are not shown all at the same time. I guess this is a style thing.